src/styles/colors.scss

```diff
@@ -0,0 +1,2 @@
+$primary: #073763;
```

Comment:
kindly rename this file to variables so that it can take any variable.

Reply:
Noted